src/test/javascript/spec/service/submission.service.spec.ts

```diff
@@ -41,28 +29,49 @@ describe('Submission Service', () => {
                 { provide: SessionStorageService, useClass: MockSyncStorage },
                 { provide: TranslateService, useClass: MockTranslateService },
             ],
-        });
+        }).compileComponents();
+
         service = TestBed.inject(SubmissionService);
         httpMock = TestBed.inject(HttpTestingController);
         expectedResult = {} as HttpResponse<Submission[]>;
+
+        submission = {
+            id: 1,
+            submitted: true,
+            type: SubmissionType.TEST,
+            text: 'Test\n\nTest\n\nTest',
+            results: [
+                {
+                    id: 2374,
+                    resultString: '1 of 12 points',
+                    score: 8,
+                    rated: true,
+                    hasFeedback: true,
+                    hasComplaint: false,
+                },
+            ],
+        };
+
+        getLatestSubmissionResult(submission)!.feedbacks = [
+            {
+                id: 2,
+                detailText: 'Feedback',
+                credits: 1,
+            },
+        ];
     });
 
-    it('should delete an existing submission', async () => {
+    it('should delete an existing submission', fakeAsync(() => {
         service.delete(187).subscribe((resp) => (expectedResult = resp.ok));
         const req = httpMock.expectOne({ method: 'DELETE' });
         req.flush({ status: 200 });
+        tick();
+
         expect(expectedResult).toBeTrue();
```

Comment:
```suggestion
        expect(expectedResult).toBe(true);
```
I know you didn’t introduce it, but this PR seems to be a good point to update it to follow the code conventions.

Reply:
Thanks for the feedback, I changed it.